packages/@aws-cdk/aws-redshift/README.md

```diff
@@ -75,4 +75,76 @@ cluster.addRotationMultiUser('MyUser', {
 });
 ```
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+## Database Resources
+
+This module allows for the creation of non-CloudFormation database resources via [custom
+resources](https://docs.aws.amazon.com/cdk/api/latest/docs/custom-resources-readme.html).
+These resources include database users and database tables.
+
+### Creating Users
+
+Create a user within a Redshift cluster database by instantiating a `User` construct. This
+will generate a username and password, store the credentials in a [Secrets Manager
+`Secret`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-secretsmanager.Secret.html),
+and make a query to the Redshift cluster to create a new database user with the
```

Comment:
Maybe add a description of how it authenticates with the Redshift cluster. Is it using the admin credentials? And this only works if you created your redshift cluster with CDK, right?

Reply:
I see later you allow them to supply credentials as well, cool.

Reply:
added description